packages/11ty/_includes/components/object-filters.webc

```diff
@@ -53,48 +32,69 @@ const humanize = (string) => {
     .join(' ');
 }
 
-const getFilterElements = () => filterNames
-  .map((filterName) => {
-    const humanizedFilterName = humanize(filterName);
-    const filterOptions = [
+const getFilters = ($data, objects) => {
+  const ignoredAttributes = ['name', 'thumbnail', 'title'];
+  const filterOrder = $data.object_filters || $data.objects.object_filters;
+  const filterNames = filterOrder
+    ? filterOrder
+    : objects.reduce((filterNames, { objectData }) => {
+      Object.entries(objectData).forEach(([key, value]) => {
+        /**
+         * Only render filters for keys:
+         * - that are not duplicates
+         * - are not ignored attributes
+         * - are either strings, numbers, or arrays of strings or numbers
+         */
+        if (
+          !filterNames.includes(key) &&
+          !ignoredAttributes.includes(key) &&
+          (
+            typeof value === 'string' ||
+            typeof value === 'number' ||
+            (Array.isArray(value) && value.every(
+              (item) => typeof item === 'string' ||
+              typeof item === 'number'
+            ))
+          )
+        ) {
+          filterNames.push(key);
+        }
+      });
+      return filterNames;
+    }, []);
+
+  return filterNames.map((item) => {
+    const fieldLabel = typeof item === 'string' ? humanize(item) : item.label;
+    const fieldName = typeof item === 'string' ? item : item.field;
+    const options = [
       { label: 'All', value: '' },
-      ...objects
-        .reduce((uniqueOptions, { objectData }) => {
-          const existingValues = uniqueOptions.map(({ value }) => value);
-          const values = !Array.isArray(objectData[filterName])
-            ? [objectData[filterName]]
-            : objectData[filterName];
-
-          values.forEach((value) => {
-            if (value && !existingValues.includes(value)) {
-              uniqueOptions.push({ label: value, value });
-            }
-          })
-
-          return uniqueOptions;
-        }, [])
-        .sort((a, b) => a.value > b.value ? 1 : -1)
+      ...objects.reduce((uniqueOptions, { objectData }) => {
+        const existingValues = uniqueOptions.map(({ value }) => value);
+        const values = !Array.isArray(objectData[fieldName])
+          ? [objectData[fieldName]]
+          : objectData[fieldName];
+
+        values.forEach((value) => {
+          if (value && !existingValues.includes(value)) {
+            uniqueOptions.push({ label: value, value });
+          }
+        })
+
+        return uniqueOptions;
+      }, [])
+      .sort((a, b) => a.value > b.value ? 1 : -1)
```

Comment:
Alternate approach destructing `value` as `a` and `b`
```suggestion
      .sort(({ value: a}, { value: b }) => a > b ? 1 : -1)
```
To be correct in all cases the comparator needs to return `0` when `a` and `b` are equivalent. 

Sorting text strings that may contain marks benefits from using `localeCompare` to correctly compare strings; `localeCompare` correctly returns `0` when the strings are equivalent.
```
const comparator = ({ value: a}, { value: b }) => a.localeCompare(b);
```

Reply:
this works with `localeCompare`, we just can't destructure + rename `a`/`b` from `{ value: a, value: b }`